CefSharp.BrowserSubprocess.Core/CefAppUnmanagedWrapper.cpp

```diff
@@ -91,64 +91,105 @@ namespace CefSharp
     {
         auto handled = false;
         auto name = message->GetName();
-        if (name == kEvaluateJavascriptRequest)
+        auto argList = message->GetArgumentList();
+
+        auto browserId = argList->GetInt(0);
+        if (browser->GetIdentifier() != browserId)
         {
-            auto argList = message->GetArgumentList();
-            auto browserId = argList->GetInt(0);
-            auto frameId = GetInt64(argList, 1);
-            auto callbackId = GetInt64(argList, 2);
-            auto script = argList->GetString(3);
+            throw gcnew InvalidOperationException(String::Format("Request BrowserId : {0} does not match browser Id : {1}", browserId, browser->GetIdentifier()));
```

Comment:
Will this crash the `BrowserProcess`? I think in general we're better to return an error response than than an a `Exception`